packages/app/src/runner/index.ts

```diff
@@ -227,7 +227,7 @@ function runSpecCT (spec: SpecFile) {
   // watches spec file for changes.
   getEventManager().setup(config)
 
-  const $runnerRoot = getRunnerElement()
+  const $runnerRoot = getRunnerElement() as HTMLElement
```

Comment:
This was because `getElementById` had an internal condition where it could sometimes not return an element (since the reporter is not guaranteed). Will address this by just removing this helper and putting the check in `getRunnerElement`.